cdap-hbase-compat-0.94/src/main/java/co/cask/cdap/data2/util/hbase/HBase94TableUtil.java

```diff
@@ -21,24 +21,78 @@
 import co.cask.cdap.data2.transaction.coprocessor.hbase94.DefaultTransactionProcessor;
 import co.cask.cdap.data2.transaction.queue.coprocessor.hbase94.DequeueScanObserver;
 import co.cask.cdap.data2.transaction.queue.coprocessor.hbase94.HBaseQueueRegionObserver;
+import co.cask.cdap.proto.Id;
+import com.google.common.base.Joiner;
 import com.google.common.collect.Maps;
+import org.apache.hadoop.conf.Configuration;
 import org.apache.hadoop.hbase.ClusterStatus;
 import org.apache.hadoop.hbase.Coprocessor;
 import org.apache.hadoop.hbase.HColumnDescriptor;
 import org.apache.hadoop.hbase.HRegionInfo;
 import org.apache.hadoop.hbase.HServerLoad;
+import org.apache.hadoop.hbase.HTableDescriptor;
 import org.apache.hadoop.hbase.ServerName;
 import org.apache.hadoop.hbase.client.HBaseAdmin;
+import org.apache.hadoop.hbase.client.HTable;
 import org.apache.hadoop.hbase.io.hfile.Compression;
 import org.apache.hadoop.hbase.regionserver.StoreFile;
 
 import java.io.IOException;
 import java.util.Map;
+import javax.annotation.Nullable;
 
 /**
  *
  */
 public class HBase94TableUtil extends HBaseTableUtil {
+
+  @Override
+  public boolean namespacesSupported() {
+    return false;
+  }
+
+  @Override
+  public HTable getHTable(Configuration conf, @Nullable String namespace, String tableName) throws IOException {
```

Comment:
When would namespace here be null?  In the case of using the default namespace?  If so, it doesn't seem like we construct the prefixed namespace correctly (would be "cdap_" instead of "cdap_default").